binscripts/rvm-installer

```diff
@@ -781,6 +781,24 @@ follow this link for details how to fix:
   fi
 }
 
+rvm_install_validate_volume_mount_mode()
+{
+  \typeset partition
+  \typeset mount_noexec_mode
+
+  partition=`df -P $rvm_path | tail -1 | cut -d' ' -f 1`
+  mount_noexec_mode=`mount | grep -q "$partition.*noexec.*"`
```

Comment:
`mount | grep -q "$partition .*noexec.*"`` - extra space to prevent matching wrong mount point